### ISSUE TITLE
Posible problem parsing amber parameter file 

### ISSUE THREAD

**Issue by cesaravila79** [Acellera/acemd_issues#34]
Hi guys,
I am trying to simulate a system using glycam ff. I use htmd for building the system and setting up the input files for equilibration.
The problem with the simulation is that it will explode at timestep 0 during minimization, so there is no time to write anything. I have checked the coordinate files and there is nothing strange. No atom bumping into another. So, I guess there is some problem in parsing the parameter file for the system. Or perhaps some problem defining the periodic cell size (although I understand this is automatically done by the script). I am attaching compressed file with a test system so you can replicate the problem on your computer. 
Cheers, 
Cesar

[test.tar.gz](https://github.com/Acellera/acemd_issues/files/876587/test.tar.gz)

**Comment by j3mdamas** [comment]
I'll try to look into it when I have some time.

**Comment by cesaravila79** [comment]
Hi Joao, 
Were you able to take a look at the error? It is driving me crazy. I have generated three different systems, 

         OMe-YuA2-VYS (GlcNS3S6S-IdoA2S0-OMe)
OMe-UYS-2uA2                            (IdoA2S0-GlcNS6S-OMe)
OMe-UYS-YuA2-VYS (GlcNS3S6S-IdoA2S0-GlcNS6S-OMe)

While the former two systems simulate without a problem, for the last one it keeps giving NaN for nonbonded energies. 

I don't know why I can add sugar residues to either 


side of YuA2 (or its equivalent 2uA2) but not to both. 

I am attaching a compressed file with the equilibration folders for each system. 
[bug.tar.gz](https://github.com/Acellera/acemd_issues/files/1223651/bug.tar.gz)

Cheers, 
Cesar

**Comment by j3mdamas** [comment]
I'm sorry Cesar, but I do not have time to look into this issue :(

**Comment by cesaravila79** [comment]
Hi Joao, 
I understand, we usually have more work than time to do it. ;-). 
Anyway, there is a bug in aceMD that is preventing it from being able to simulate this system. 

I have used tleap from both HTMD and AmberTools16 to build the  parameter file and the resulting file is exactly the same (except for the timestamp). This leaves out a problem in tleap. 

Just to discard any effect from water or ions overlaps, and for testing purposes, I have built the system for just the molecule in vacuum. 

Now, when I tried to minimize the system with acemd the nonbonded energy goes up to NaN while for the same files, minimization works with sander and the nonbonded energies give a reasonable number.

All the files can be found in here, in case @mj-harvey  wants to take a look ;-). 
 
[bug2.tar.gz](https://github.com/Acellera/acemd_issues/files/1235476/bug2.tar.gz)

Cheers, 
Cesar
